priv/templates/phx.gen.html/index.html.heex

```diff
@@ -1,8 +1,8 @@
 <.header>
   Listing <%= schema.human_plural %>
   <:actions>
-    <.link href={~p"<%= schema.route_prefix %>/new"}>
-      <.button>New <%= schema.human_singular %></.button>
+    <.link href={~p"<%= schema.route_prefix %>/new"} class="py-2 px-3 text-sm font-semibold rounded-lg bg-zinc-100 hover:bg-zinc-200/80">
```

Comment:
I suggest this style since a very similar style is already used for the button-alike link "Get Started" in app.html.heex

Reply:
We should avoid using tailwind styles in generators. The home page is fine, because we discard it. Could we use <.button> only instead?

Reply:
In my projects I have a `<.button link={%{patch: ...}}` that then has a properly styled `<.link` inside. Maybe that's the way to go?

Reply:
What about plain old link by using simply <.link> without any fancy style?
Buttons would always require javascript and some users might create their projects with --no-live flag.

Or the generators could produce different elements:
-  mix phx.gen.live could use <.button> with phx-click only
- while mix phx.gen.html could stick to simple <.link> (which could still be styled/replaced further if CoreComponents gets some button-alike link component in the future)